pyrometheus/chem_expr.py

```diff
@@ -280,13 +280,12 @@ def equilibrium_constants_expr(sol: ct.Solution, reaction_index, gibbs_rt):
             for indices_prod_i, nu_prod_i in zip(indices_prod, nu_prod))
 
     # Check if reaction is termolecular
-    sum_nu_net = sum(nu_prod) - sum(nu_reac)
+    sum_nu_net = sum(nu_reac) - sum(nu_prod)
```

Comment:
Any reason for sign changes here and line 285?

Reply:
Obsessive compulsive disorder. I dislike "- -1.0" expressions.

Reply:
Understood, but `nu_net` often appears in references as `nu_prod - `nu_reac`, so let's keep it as such.